src/client/Microsoft.Identity.Client/TelemetryCore/Internal/Events/EventBase.cs

```diff
@@ -23,14 +22,13 @@ internal abstract class EventBase : Dictionary<string, string>
         private readonly long _startTimestamp;
         public const string TenantPlaceHolder = "<tenant>"; // It is used to replace the real tenant in telemetry info
 
-        public EventBase(string eventName, string correlationId)
+        public EventBase(string eventName)
```

Comment:
Is this base class necessary after all the refactoring? We only have ApiEvent? In paticular, the `ScrubTenant` method seems to do a lot of work.